src/compute-client/src/controller/instance.rs

```diff
@@ -219,6 +221,16 @@ pub(super) struct Instance<T> {
     wallclock_lag: WallclockLagFn<T>,
     /// Dynamic system configuration.
     dyncfg: Arc<ConfigSet>,
+
+    /// Sender for updates to collection read holds.
+    ///
+    /// Copies of this sender are given to [`ReadHold`]s that are created in
+    /// [`Instance::acquire_read_hold`].
+    read_holds_tx: mpsc::UnboundedSender<(GlobalId, ChangeBatch<T>)>,
```

Comment:
There aren't any `async` usages right? If so we can just use std's channels here

Reply:
`ReadHold::new` expects a `tokio::sync::mpsc::UnboundedSender`, so we need to use one here too. It's also used by storage, which listens on the receiver in a `select!` so we can't just change the channel type.